Server/App/api/deck.py

```diff
@@ -85,15 +85,17 @@ def put(self, name, public, tags, actions, parent_deck_version, parent_user_data
                 deck.set_name(name)
             if public is not None:
                 deck.set_public(public)
+            if unshare is True:
+                deck.nullify_share_code()
```

Comment:
Looks like anyone who added the deck prior to it being unshared will still be able to fetch the newest versions of the deck. Is this intended?